apiserver/common/modelstatus.go

```diff
@@ -52,10 +52,12 @@ func (c *ModelStatusAPI) modelStatus(tag string) (params.ModelStatus, error) {
 	st := c.backend
 	if modelTag != c.backend.ModelTag() {
 		otherSt, releaser, err := c.backend.GetBackend(modelTag.Id())
+		if releaser != nil {
```

Comment:
This isn't quite right - releaser will never be non-nil if err != nil
So the original code is ok here.
What does need to change though is the code inside this method
func (st modelManagerStateShim) GetBackend(modelUUID string) (ModelManagerBackend, func() bool, error)